Detectors/TRD/base/test/testRawData.cxx

```diff
@@ -82,8 +82,8 @@ BOOST_AUTO_TEST_CASE(TRDRawDataHeaderInternals)
   mcmrawdataheader.word = 0x01fe;
   BOOST_CHECK_EQUAL(mcmrawdataheader.pid0, 0xff); // 8 bits
   //check tracklet
-  tracklet.word = 0xffc00000;
-  BOOST_CHECK_EQUAL((uint32_t)tracklet.pos, 0x3ff);
+  tracklet.word = 0xffe00000;
+  BOOST_CHECK_EQUAL((uint32_t)tracklet.pos, 0x7ff);
```

Comment:
Which format is this tracklet word here? I don't understand the 0xffe00000, but probably this is fine

Reply:
maybe i should name it better, its the tracklet "portion" of the raw data TrackletMCMData 1 to 3 per TrackletMCMHeader, and word is the raw form of the TrackletMCMData which is merely a 32 bit integer, so that ffe is the full scale of the position.